app/assets/stylesheets/welcome.scss

```diff
@@ -140,6 +127,33 @@
   }
 }
 
+#right-panel {
+  position:relative;
+  left: -0.6%;
```

Comment:
if you want to create margin between 2 elements, please use 
```css
margin-left or margin-right: few pixels
```
percents here is not useful.